README.md

```diff
@@ -228,11 +225,14 @@ not found states.
 
 Backbone | Ember | React | UI-Router | Stateman | StateRouter
 -------- | ----- | ----- | --------- | -------- | -----------
-✔        | ✘     | ✔     | ✘         | ✘        | ✘
+✔        | ✘     | ✔     | ✘         | ✔        | ✘
```

Comment:
Heh, yeah. I noticed the error after I hit merge, then [fixed it after](https://github.com/jmeas/router-comparison/commit/5d90a481eef43e8c56a94b084760306ae79c8462#diff-04c6e90faac2675aa89e2176d2eec7d8L228).